selfdrive/test/test_onroad.py

```diff
@@ -8,7 +8,7 @@
 import subprocess
 import time
 import numpy as np
-import zstd
+import zstandard as zstd
```

Comment:
Says you can create a zstd context to compress in tight loops, but I didn't notice a huge difference in our case.